### PR TITLE
solve compilation problem for iOS7 targeted apps

### PR DESCRIPTION
**Pull request by jankase**
Solve issue when the code is included in iOS 7 targeted app using Swift 2.0

### CODE REVIEW

File: Pod/Classes/LocksmithRequest.swift [hunk 1]
```diff
@@ -83,7 +90,11 @@ public enum Accessible: RawRepresentable {
         case .Always:
             return String(kSecAttrAccessibleAlways)
         case .WhenPasscodeSetThisDeviceOnly:
-            return String(kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly)
+            if #available(iOS 8.0, *) {
+                return String(kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly)
+            } else {
+                fatalError("This value has no raw representation in your iOS 7")
```
Comment (matthewpalmer) [inline Pod/Classes/LocksmithRequest.swift]: I think we should change this error message to something like `"Accessible.WhenPasscodeSetThisDeviceOnly has no raw representation in iOS 7."`

Thoughts?

Comment (jankase) [inline Pod/Classes/LocksmithRequest.swift]: I agree this is much better text. 

I thought again about the solution and I find way how to implement it better to assure that this message is only for compiler to be happy and in reality the user never reach this situation. See my second commit.

Jan

**Comment by matthewpalmer** [comment]
Thanks very much! 👍

**Comment by jankase** [comment]
np

**Pull request merged**